### PR TITLE
[FLINK-7017] Remove netty usages in flink-tests

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/WebFrontendITCase.java [hunk 1]
```diff
@@ -83,16 +78,16 @@ public static void initialize() throws Exception {
...
 		Files.createFile(outFile.toPath());
-		
+
 		config.setString(JobManagerOptions.WEB_LOG_PATH, logFile.getAbsolutePath());
 		config.setString(ConfigConstants.TASK_MANAGER_LOG_PATH_KEY, logFile.getAbsolutePath());
 
 		cluster = new LocalFlinkMiniCluster(config, false);
 		cluster.start();
-		
+
```
Comment (zhangminglei) [inline flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/WebFrontendITCase.java]: Sometimes, I also get this kinda stuff. But seems i didnt do anything relevant to this style.

Comment (zentol) [inline flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/WebFrontendITCase.java]: this is the removal of leading tabs that is required by the checkstyle active in `flink-runtime-web`.

**Comment by greghogan** [comment]
LGTM.

**Comment by zentol** [comment]
merging.

**Pull request closed without merging**